### PR TITLE
Web Apps Presentation

### PR DESCRIPTION
**Pull request by waxenegger**
The presentation from my private repo: https://github.com/waxenegger/frameworks/tree/master/docs

I have kept the directory structure as is (except using reveal.js from the project's template directory)

Note: presentation can be viewed like this (adjust path param for 'root' directory):

```
$ jekyll serve .
```

### CODE REVIEW

**Comment by sbesson** [comment]
What is the difference between the embedded version of `reveal-js` and the one at https://github.com/ome/presentations/tree/master/template/reveal-js ?

**Comment by waxenegger** [comment]
I'd have to check but I think none. I believe I started from the template, what I did change was some css in the index.html and that I'd rather not have combined with another presentation's css but otherwise I don't think there should be a problem using the template directory's reveal.js.

**Comment by sbesson** [comment]
:+1: can you actually combine both commits into one and force push as this will prevent the addition followed by the removal of all these files in the repo.

**Comment by waxenegger** [comment]
I think that should have done the job.

File: 2017/Team-Training/OMERO-apps/index.html [hunk 1]
```diff
@@ -0,0 +1,1603 @@
...
+                <!-- app setup - static/templates/index.html -->
+                <section>
+                    <h2><i>app/templates/app/index.html</i></h2>
+                    <br/>
+                    <pre style="box-shadow: none;">
+                        <code style="max-height: 450px" class="xml">
+&lt;html&gt;
+    &lt;head&gt;
+      &lt;link rel="stylesheet" type="text/css"
+              href=" {% static 'app_directoryr/css/app.css?version=1.1' %}" /&gt;
```
Comment (hflynn) [inline 2017/Team-Training/OMERO-apps/index.html]: The 'r' in 'app_directoryr/css/...' here looks to be a typo

File: 2017/Team-Training/OMERO-apps/index.html [hunk 2]
```diff
@@ -0,0 +1,1603 @@
...
+                    <h3>View - right_panel_view.js</h3>
+                    <div style="float: left;margin-top: 150px;width:250px">
+                        <img src="images/figure_view_changes.png"/>
+                    </div>
+                    <div style="float: left;width: 800px">
+                        <pre style="box-shadow: none;margin: 5px;">
+                            <code style="min-height: 600px"
+                                  class="javascript">
+// The 'Right Panel' is the floating Info, Preview etc display.
+// It listens to selection changes on the FigureModel and updates it's display
```
Comment (hflynn) [inline 2017/Team-Training/OMERO-apps/index.html]: it's -> its

File: 2017/Team-Training/OMERO-apps/index.html [hunk 3]
```diff
@@ -0,0 +1,1603 @@
...
+// By creating new Sub-Views
+
+var RightPanelView = Backbone.View.extend({
+
+  initialize: function(opts) {
+      // we render on selection Changes in the model
```
Comment (hflynn) [inline 2017/Team-Training/OMERO-apps/index.html]: 'Changes' doesn't need to have an initial capital, does it?

File: 2017/Team-Training/OMERO-apps/index.html [hunk 4]
```diff
@@ -0,0 +1,1603 @@
...
+ * @classdesc
+ * Ellipse is an extension of the built-in ol.geom.Polygon that will allow
+ * you to create ellipses within the open layers framework.
+ *
+ * At present the approach taken is aiming at best integration into openlayers
+ * as well as cross browser support. For HTMLCanvas there is a drawEllipse
+ * method which, toDate, is only supported by Chrome.
+ *
+ * That said, there are various methods out there how people accomplish the task
+ * of drawing en ellipse on an HTMLCanvas. They range from scaled circles, over
```
Comment (hflynn) [inline 2017/Team-Training/OMERO-apps/index.html]: en -> an

**Review by hflynn — COMMENTED** [review]
A few nitpicks but probably the 1st (probable) typo is the only one that isn't copy-pasted from the codebase and therefore the only one that really needs fixed.

File: 2017/Team-Training/OMERO-apps/index.html (hunk 1 above)
Comment (waxenegger) [reply 2017/Team-Training/OMERO-apps/index.html]: yes, thank you

**Review by hflynn — APPROVED** [review]
All sorted thanks, good to merge

**Comment by sbesson** [comment]
Deployed live at https://downloads.openmicroscopy.org/presentations/2017/Team-Training/OMERO-apps/

**Pull request merged**